### PR TITLE
Fixes #26839 - added dns_view option

### PR DESCRIPTION
**Pull request by lzap**
I am pushing without testing this as my `rake spec` fails about 90% of all tests. This is a new install, Ruby 2.6, `bundle install` did not work as it wanted to upgrade `beaker` gem which I upgraded and then all tests were failing.

### CODE REVIEW

**Comment by lzap** [comment]
Fixed: manifests/plugin/dns/infoblox/params.pp:5:WARNING: double quoted string containing no variables

**Comment by lzap** [comment]
Fixed: manifests/plugin/dns/infoblox.pp:17:WARNING: missing documentation for class parameter

File: manifests/plugin/dns/infoblox/params.pp [hunk 1]
```diff
@@ -2,4 +2,5 @@
   $dns_server = undef
   $username   = undef
   $password   = undef
+  $password   = 'default'
```
Comment (ekohl) [inline manifests/plugin/dns/infoblox/params.pp]:
```suggestion
  $dns_view   = 'default'
```

File: manifests/plugin/dns/infoblox.pp [hunk 2]
```diff
@@ -10,10 +10,13 @@
 #
 # $password::   The password of the Infoblox user
 #
+# $dns_view::   The DNS Infoblox view
```
Comment (pgagne) [inline manifests/plugin/dns/infoblox.pp]: Maybe change this to "The Infoblox DNS View"

**Comment by lzap** [comment]
Amended changes.

File: spec/classes/foreman_proxy__plugin__dns__infoblox_spec.rb [hunk 3]
```diff
@@ -30,6 +31,7 @@
         ':dns_server: "localhost"',
         ':username: "admin"',
         ':password: "infoblox"',
+        ':password: "default"',
```
Comment (ekohl) [inline spec/classes/foreman_proxy__plugin__dns__infoblox_spec.rb]:
```suggestion
        ':dns_view: "default"',
```

Comment (lzap) [reply spec/classes/foreman_proxy__plugin__dns__infoblox_spec.rb]: Damn I am poor developer. Fixed.

**Comment by lzap** [comment]
All hopefully resolved.

**Comment by mmoll** [comment]
merged, díky @lzap!

**Pull request merged**